test/logics/radiant-v2/borrow.test.ts

```diff
@@ -0,0 +1,73 @@
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/signers';
+import { claimToken, getChainId, snapshotAndRevertEach } from '@protocolink/test-helpers';
+import * as common from '@protocolink/common';
+import * as core from '@protocolink/core';
+import { expect } from 'chai';
+import * as helpers from './helpers';
+import hre from 'hardhat';
+import * as radiantv2 from 'src/logics/radiant-v2';
+
+describe('arbitrum: Test RadiantV2 Borrow Logic', () => {
+  let chainId: number;
+  let users: SignerWithAddress[];
+
+  before(async () => {
+    chainId = await getChainId();
+    const [, user1, user2] = await hre.ethers.getSigners();
+    users = [user1, user2];
+    await claimToken(chainId, user1.address, radiantv2.arbitrumTokens.USDC, '20000');
+    await claimToken(chainId, user2.address, radiantv2.arbitrumTokens.WETH, '100');
+  });
+
+  snapshotAndRevertEach();
+
+  const testCases = [
+    {
+      userIndex: 0,
+      deposit: new common.TokenAmount(radiantv2.arbitrumTokens.USDC, '5000'),
+      output: new common.TokenAmount(radiantv2.arbitrumTokens.WETH, '1'),
+      interestRateMode: radiantv2.InterestRateMode.variable,
```

Comment:
Should we add tests for `InterestRateMode.stable`?

Reply:
Due to Radiant v2 only support stable mode (on UI and their doc), the stable mode tests here may not be essential